array-manipulation-2/src/chunk.js

```diff
@@ -0,0 +1,36 @@
+/* exported chunk */
+/*
+  set output equal to empty array
+  set chunk equal to empty array
+  set chunkCounter equal to zero
+  set i equal to length of array
+    while i is less than length of array
+      append array at i to chunk
+      increment chunkcounter by one
+      if chunkCounter is euqal to size
+        push chunk to output
+    if chunk length greater thank zero
+      append chunk to output
+    return output
+*/
+
+function chunk(array,size){
+  debugger;
```

Comment:
Please remove `debugger;` statements prior to submitting code for review. Good to see you using it, though!!